src/validations/rules/sap.sch

```diff
@@ -69,6 +70,25 @@
                 id="has-import-ssp"
                 role="error"
                 test="oscal:import-ssp">An OSCAL SAP must have an import-ssp element.</sch:assert>
+                
+            <sch:let
+                name="databases"
+                value="
+                $ssp-doc//oscal:component[oscal:prop[@name = 'type' and @value eq 'database']]/@uuid  |
+                $ssp-doc//oscal:inventory-item[oscal:prop[@name = 'type' and @value eq 'database']]/@uuid |
+                //oscal:local-definitions/oscal:activity[oscal:prop[@value eq 'database']]/@uuid" />
+            <sch:let name="sap-database-tasks"
+                value="//oscal:task[oscal:prop[@value='database']]/oscal:associated-activity/@activity-uuid ! xs:string(.)"/>
+            <sch:let name="missing-database-tasks"
+                value="$databases[not(. = $sap-database-tasks)]"/>
+            <sch:assert
+                diagnostics="has-database-diagnostic"
+                doc:guide-reference="Guide to OSCAL-based FedRAMP Security Assessment Plans (SAP) §4.6"
+                fedramp:specific="true()"
+                id="has-database"
+                role="error"
+                test="count($databases[not(. = $sap-database-tasks)]) = 0"
+                unit:override-xspec="both">For every database to be tested there must be a matching task entry.</sch:assert>
```

Comment:
Formatting and indents are likely due to conflict resolution on git rather than IDE.  I expect I am going to need a quick issue to fix format, indents, reorder for clarity after I have worked through the Guide.

I agree it is lacking.  I located matching uuid values in the SAP and SSP and made the connections.  That was probably ill-advised.  

I'd like to retract this PR and place the entire issue into a holding pattern until we hear from the PMO with guidance.  Do you agree?